server/routes/settings.js

```diff
@@ -0,0 +1,134 @@
+const express = require("express");
+const router = express.Router();
+const mongoose = require("mongoose");
+const SettingsModel = require("./../models/Settings");
+const UserModel = require("./../models/User");
+const { jwtVerify } = require("../utils/authUtils");
+
+mongoose.connect(process.env.MONGO_URL, { useNewUrlParser: true  });
+
+const saveSettings = (settings, res) => {
+  settings.save((err) => {
+    if (err) {
+      console.log(err);
+    }
+  });
+  res.send({ error: false, settings });
+}
+
+router.post("/settings/:email", jwtVerify, (req, res, next) => {
+  UserModel.findOne({ username: req.params.email },
+    (err, user) => {
+      if (err) {
+        console.log(err);
+      }
+      if (user) {
+        SettingsModel.findOne({ email: req.params.email },
+          (err, settings) => {
+            if (err) {
+              console.log(err);
+            }
+            if (settings) {
+              res.send({ error: true });
```

Comment:
Here, for example, you can just call next('User already exists') and the error handler will do a res.send by itself. The only argument would just be the error message string you want.

Reply:
Alright, I changed to use next()